contents/performance/PerformanceProfiling.md

```diff
@@ -0,0 +1,129 @@
+# Performance Profiling
+
+Author: [Ong Heng Le](https://github.com/initialshl)
+
+## Overview
+
+A performance profiler is a tool which collects data (such as function timings) about 
+your program to identify the areas with performance issues, commonly used in code 
+optimization. CPU sampling and instrumentation are the two most common types of 
```

Comment:
Can be more concise:

~~CPU sampling and instrumentation~~ There are ~~the~~ two most common types of performance profiling methods: